packages/local-cli/commands/serve.js

```diff
@@ -20,6 +20,8 @@ module.exports = function serveCommand (callback) {
       if (argv.production) {
         process.env.NODE_ENV = 'production';
       }
+      // production environment always refers to starting dynamic mode
```

Comment:
I'm afraid that's not really true - Hops does, in fact support, static production mode. For that case, we even have some specific code in place to handle expires headers for pre-generated HTML files: https://github.com/xing/hops/blob/master/packages/express/app.js#L20